docs/source/conf.py

```diff
@@ -93,6 +98,8 @@
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = "sphinx"
 
+myst_gfm_only = True  # Enable GitHub flavored markdown
```

Comment:
This leads to
```
ModuleNotFoundError: Linkify enabled but not installed.
```

Reply:
oh yes, needs to be added to pyproject.toml

Reply:
removed the GitHub flavored markdown